src/main/java/com/werfen/report/service/pdf/util/PdfExportService.java

```diff
@@ -23,6 +24,19 @@ public File export(JasperPrint jasperPrint, String filePath) throws JRException
         return new File(filePath);
     }
 
+    public ByteArrayOutputStream exportToStream(JasperPrint jasperPrint) throws JRException {
```

Comment:
I would suggest to move the shared part of this new method (usage of JRPdfExporter) into a new private auxiliary method, so we can avoid some duplication. I would suggest something like:
```java
private void exportDocument(JasperPrint jasperPrint, SimpleOutputStreamExporterOutput outputStream)
```